### ISSUE TITLE
os::windows::snmp::plugin --mode=storage shows different output between CLI and GUI

### ISSUE THREAD

**Issue by LemonLight** [centreon/centreon-plugins#3523]
Hello everyone,

I have some trouble by configuring my windows server disks monitoring. 

Context : I have a VM with 2 disk that I want to monitor separately (C and D)

In CLI, my command is:
`perl centreon_plugins.pl --plugin=os::windows::snmp::plugin --mode=storage --hostname=myhost--snmp-version=2c --snmp-community=myCommunity --warning-usage=80 --critical-usage=90 --storage='(^C)' --regexp --name`

In CLI I have the following output:

- For C : 
`OK: Storage 'C:\ Label:System  Serial Number a76f2be' Usage Total: 58.79 GB Used: 24.02 GB (40.86%) Free: 34.77 GB (59.14%) | 'count'=1;;;0; 'used'=25791307776B;0:50499416883;0:56811843993;0;63124271104`

- For D : 
`CRITICAL: Storage 'D:\ Label:Applis  Serial Number ec80e4ee' Usage Total: 1006.00 MB Used: 970.51 MB (96.47%) Free: 35.48 MB (3.53%) | 'count'=1;;;0; 'used'=1017655296B;0:843890688;0:949377024;0;1054863360`

Then in GUI I have the the following command linked to my service template:
`$CENTREONPLUGINS$/centreon-plugins/centreon_plugins.pl --plugin=os::windows::snmp::plugin --mode=storage –storage='(^$ARG1$)' --hostname=$HOSTADDRESS$ --snmp-version=$_HOSTSNMPVERSION$ --snmp-community=$_HOSTSNMPCOMMUNITY$ --warning-usage=$ARG2$ --critical-usage=$ARG3$ --regexp --name`

With the following result for both services:

- Status information:	
`CRITICAL: Storage 'D:\ Label:Applis Serial Number ec80e4ee' Usage Total: 1006.00 MB Used: 970.51 MB (96.47%) Free: 35.48 MB (3.53%)`
- Performance data: 
`'count'=2;;;0;
'used_C:\ Label:System  Serial Number a76f2be'=25789820928B;0:50499416883;0:56811843993;0;63124271104
'used_D:\ Label:Applis  Serial Number ec80e4ee'=1017655296B;0:843890688;0:949377024;0;1054863360`

Sounds weird doesn't it ?

Any idea ? :) 
Thanks in advance !

**Comment by garnier-quentin** [comment]
I don't see the issue. In perfdata it's in Bytes (multiply the output MB by: 1024 * 1024)

**Comment by LemonLight** [comment]
Hello Quentin, 

The purpose is to create a service template to check the disk by using its letter in Windows. 
Howerver, when we were configuring it, all disks of the servers were shown instead of just the letter we defined in the service template.

In the diagnostic, we supposed our Centreon and centreon-plugin version were a little bit old and we started to update. 
Then, after upgrade and some adjustment in our check command and service template, it has finally worked perfectly :) 

Check command : 
`$CENTREONPLUGINS$/centreon-plugins/centreon_plugins.pl --plugin=os::windows::snmp::plugin --mode=storage --storage=$ARG1$ --hostname=$HOSTADDRESS$ --snmp-version=$_HOSTSNMPVERSION$ --snmp-community=$_HOSTSNMPCOMMUNITY$ --warning-usage=$ARG2$ --critical-usage=$ARG3$ --regexp --name`

Service template $ARG1$ value = '(^C)'

Problem solved :)
